documentation/RLC-quickstart.md

```diff
@@ -1,68 +1,73 @@
+
+1. Project Folder Structure.  
+   Normally, the folder structure would be something like `sdk/{servicename}/{servicename}-{modulename}-rest`. For example, we have `sdk/agrifood/agrifood-farming-rest` folder for Farmbeats account modules under {SDK_REPO_ROOT}. That folder will be your **${PROJECT_ROOT} folder**.  
 1. Package Name Convention.  
    The package name for RLC is something like `@azure-rest/{servicename}-{modulename}`. For example, the package name for Farmbeats module is `@azure-rest/agrifood-farming`.
 
+
 # How to generate RLC
 
 We are working on to automatically generate everything right now, but currently we still need some manual work to get a releasable package. Here're the steps of how to get the package.
 
-1. **Create a swagger/README.md file.under ${PROJECT_ROOT} folder**  
-    We are using autorest to generate the code, but there's a lot of command options and in order to make the regenerate process easier in the cases of refresh the rest api input or change the code generator version, you need to document the generate command parameters.  
-    Here's an example of the swagger/README.md
+1. **Add typescript emitter dependency in package.json**  
+   
+   In Cadl project, modify `package.json` to add dependency for typescript emitter, then run `npm install` again to install `@azure-tools/cadl-typescript`.
 
-    ~~~
-    
-    # Azure  Farmbeats TypeScript Protocol Layer
-    
-    > see https://aka.ms/autorest
-    ## Configuration
+   ```json
+   "dependencies": {
+     ...
+     "@azure-tools/cadl-typescript": "1.0.0-beta.3"
+   },
+   ```
+
+   ---
+   **NOTE**
+    It's always recommended to replace the version of emitter cadl-typescript with the latest version you can find in [npmjs.com](https://www.npmjs.com/package/@azure-tools/cadl-typescript) in latest tag.
+
+   ---
+
+1. **Configure typescript emitter in cadl-project.yaml**
+
+    In Cadl project, modify (or create) `cadl-project.yaml` and configure the SDK generated, using the emitter options on `@azure-tools/cadl-typescript`
     
     ```yaml
-    package-name: "@azure-rest/agrifood-farming"
-    title: Farmbeats
-    description: Farmbeats Client
-    generate-metadata: true
-    generate-test: true
-    license-header: MICROSOFT_MIT_NO_VERSION
-    output-folder: ../
-    source-code-folder-path: ./src
-    input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/683e3f4849ee1d84629d0d0fa17789e80a9cee08/specification/agfood/data-plane/Microsoft.AgFoodPlatform/preview/2021-03-31-preview/agfood.json
-    package-version: 1.0.0-beta.2
-    rest-level-client: true
-    security: AADToken
-    security-scopes: https://farmbeats.azure.net/.default
-    use-extension:
-      "@autorest/typescript": "6.0.0-rc.1"
+    emitters:
+      "@azure-tools/cadl-typescript":
```

Comment:
Is there any other options like output dir that service team need to configure?

Reply:
We have no output dir option but we do have some other options. We add a link to it : https://github.com/Azure/autorest.typescript/tree/main/packages/cadl-typescript#emitter-options

Reply:
For now, there is a propery `sdk-folder` (value must be absolute path) and the generated codes will be that path.